spec/money_spec.rb

```diff
@@ -618,6 +618,16 @@
     end
   end
 
+  describe "calculate_splits" do
+    specify "#calculate_splits gives 1 cent to both people if we start with 2" do
+      number_high, number_low, high, low = Money.new(0.02).calculate_splits(2)
+      expect(number_high).to eq(0)
+      expect(number_low).to eq(2)
+      expect(high.value).to eq(0.02)
+      expect(low.value).to eq(0.01)
```

Comment:
Assuming the currency will be CAD, why not `expect(low).to eq(Money.new(0.01, "CAD"))`?

Reply:
Done :)